modules/connected-to-disconnected-config-registry.adoc

```diff
@@ -0,0 +1,182 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/connected-to-disconnected.adoc
+
+[id="connected-to-disconnected-config-registry_{context}"]
+= Configuring the mirror registry
+
+After creating the mirror registry and mirroring the images to the registry, you need to make some modifications to your cluster in order for pods to be able to pull images from the mirror registry. 
+ 
+You must add the mirror registry credentials to the global pull secret, add the mirror registry server certificate to the cluster, and create an `ImageContentSourcePolicy` custom resource that associates the mirror registry with the source registry.
+
+. Add mirror registry credential to cluster global pull-secret:
++
+[source,terminal]
+----
+$ oc set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson=<pull_secret_location> <1>
+----
+<1> Provide the path to the new pull secret file.
++
+For example:
++
+[source,terminal]
+----
+$ oc set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson=.dockerconfigjson
```

Comment:
To avoid confusion, I think we should not use `.dockerconfigjson` twice, we can maybe use something like `.mirrorsecretconfigjson` for the mirror registry pull secret file name.